ssas/groups.go

```diff
@@ -23,7 +24,7 @@ type SystemSummary struct {
 	GID        uint      `json:"-"`
 	ClientName string    `json:"client_name"`
 	ClientID   string    `json:"client_id"`
-	IPs        []string  `json:"ips,omitempty"`
+	IPs        []string  `json:"ips,omitempty" gorm:"-"`
```

Comment:
Had to change because gorm was returning an error around not being able to handle the IPs field since it's not defined in the Systems table.

We can tie a foreign key between System and IPs tables, but leaving it out for now since it was not being done with gorm v1.